build/pause/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 3.10
+* Support both 1809 and ltsc2019. 1809 will be deprecated soon. ([#116935](https://github.com/kubernetes/kubernetes/pull/116935), [@lzhecheng](https://github.com/lzhecheng))
```

Comment:
Nit: for additional clarity, you could mention `` Support for both Windows 1809 and ltsc2019``.

Reply:
Updated